src/chapters/angular-lsn1/studio.rst

```diff
@@ -1,62 +1,197 @@
+   * You should see a header that says "Welcome to mission-planning-dashboard!"
 
-Project Data
+10. Stage and commit the files before starting on the features.
+
+Requirements
 ------------
+The mission dashboard you are creating will eventually look like this.
 
-Data you will need for this project:
-   #. Mission name: Apollo 11
-   #. Rocket name: Saturn V
-   #. Crew member names & roles: Commander Neil Armstrong, Command Module Pilot
-      Michael Collins, Lunar Module Pilot Buzz Aldrin
-   #. Equipment: Command Module, Lunar Module, 3 flight suits, 2 moon suits, 30
-      meals, 100 litres of water
-   #. Countries: United States
+.. figure:: ./figures/example-mission-dashboard.png
+   :alt: Screen shot showing the mission dashboard with mission name, rocket name, crew members, equipment, and experiments.
 
-If you need a reminder on how to create components and organize them using the
-app component work through the :ref: `` again.
+Update Starter Page Content
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+The default stater page created by Angular contains default text, images, and links. Your job is to remove the default content.
 
-Bonus Mission
--------------
+1. Clear out the contents of file ``mission-planning-dashboard/src/app/app.component.html``.
+2. Type in the text ``Add components here...`` into ``app.component.html``.
+3. View the app in your browser to verify the words "Add components here..." is the only thing that appears on the page.
+
+Header Component
+^^^^^^^^^^^^^^^^
+You need to create a component that shows the title, mission name, and carrier rocket.
+
+1. Create a header component using ``ng g component header``.
+2. In file ``/mission-planning-dashboard/src/app/header/header.component.html`` add HTML:
+
+.. sourcecode:: html+ng2
+   :linenos:
+
+   <h1>Mission Planning Dashboard</h1>
+   <h3>Mission Name: {{ missionName }}</h3>
+   <h3>Carrier Rocket: {{ rocketName }}</h3>
+
+3. Add variables ``missionName`` and ``rocketName`` to the header component in ``mission-planning-dashboard/src/app/header/header.component.ts``
+
+.. sourcecode:: TypeScript
+
+   export class HeaderComponent implements OnInit {
+
+       missionName: string = "Mars 2030";
+       rocketName: string = "Plasma Max";
+
+4. Add a reference to the header component in ``mission-planning-dashboard/src/app/app.component.html``.
+
+.. sourcecode:: html+ng2
+
+   <app-header></app-header>
+
+
+5. View the app in your browser to verify that the title, mission name, and rocket name are visible.
+
+.. figure:: ./figures/example-header-component.png
+   :alt: Screen shot of browser showing address localhost:4200, which has a title of Mission Planning Dashboard, a Mission Name, and a Carrier Rocket.
+
+Crew Component
+^^^^^^^^^^^^^^
+Next you need to make a component to show a list of crew members.
+
+1. Create the component by running ``ng g component crew``.
+2. Set the contents of to be:
+
+.. sourcecode:: html
+
+   <h3>Crew</h3>
+   <ul>
```

Comment:
The students have been introduced to placeholders. How about having the students use them in the list elements, with supporting data in ``crew.component.ts``?

Reply:
I'll add that as a bonus item